admission-webhook/deploy/.helpers.sh

```diff
@@ -30,10 +30,9 @@ fatal_error() {
     exit 1
 }
 
-if [ ! "$KUBECTL" ]; then
-    KUBECTL=$(which kubectl) || true
-fi
-if [ ! -x $KUBECTL ]; then
+KUBECTL=$(which kubectl) || true
```

Comment:
I'm not sure I get the idea here. In any case, this is what causes CI to fail: KUBECTL is set by .travis.sh, as well as in the Makefile for the normal dev workflow.

Reply:
During the tests, KUBECTL is set to "./cluster/kubectl.sh", but since this script runs in a different PWD, that path is not available, and this makes the test fail.
Resetting KUBECTL seemed like the easiest fix, because I assumed that kubectl will always be in the PATH, but the travis test fails because it's not. If you don't think it's acceptable to add kubectl to PATH in the travis test, I will need to rethink the fix for this.

Reply:
Thanks for the explanation.

Would it be at all possible to set KUBECTL to an absolute path instead?

Otherwise, how about keeping `$KUBECTL` as is _if it points to an executable_, otherwise try to get it from the PATH, otherwise fail?

Reply:
Unfortunately it's set from upstream code: https://github.com/kubernetes/test-infra/blob/d5f7be8edd4e3ee6885b4857d7240eaeb1c3ff61/kubetest/e2e.go#L56
Since I don't currently know the full implications of changing this to an absolute path, I'd prefer to go with the second option.

Reply:
👍

Reply:
Pushed 3c2456f